azure-pipelines.yml

```diff
@@ -4,16 +4,101 @@
 
 trigger:
-- main
+  branches:
+    include:
+      - '*'
 
 pool:
   vmImage: 'ubuntu-20.04'
 
+container:
+  image: sonicdev-microsoft.azurecr.io:443/sonic-slave-buster:latest
+
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+- task: DownloadPipelineArtifact@2
+  inputs:
+    source: specific
+    project: build
+    pipeline: 1
+    artifact: sonic-buildimage.kvm
+    runVersion: 'latestFromBranch'
+    runBranch: 'refs/heads/master'
+  displayName: "Download artifacts from latest sonic-buildimage build"
+
+- script: |
+    set -xe
+    sudo dpkg -i libnl-3-200_*.deb
+    sudo dpkg -i libnl-genl-3-200_*.deb
+    sudo dpkg -i libnl-route-3-200_*.deb
+    sudo dpkg -i libnl-nf-3-200_*.deb
+    sudo dpkg -i libhiredis0.14_*.deb
+    sudo dpkg -i libswsscommon_1.0.0_amd64.deb
+    sudo dpkg -i python-swsscommon_1.0.0_amd64.deb
+    sudo dpkg -i python3-swsscommon_1.0.0_amd64.deb
+  workingDirectory: $(Pipeline.Workspace)/target/debs/buster/
+  displayName: 'Install Debian dependencies'
+
+- script: |
+    set -xe
+    pip2 install swsssdk-2.0.1-py2-none-any.whl
+    pip2 install sonic_py_common-1.0-py2-none-any.whl
+    pip3 install swsssdk-2.0.1-py3-none-any.whl
+    pip3 install sonic_py_common-1.0-py3-none-any.whl
+  workingDirectory: $(Pipeline.Workspace)/target/python-wheels/
+  displayName: 'Install Python dependencies'
+
+- script: |
+    set -xe
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-chassisd
+  displayName: 'Test and build sonic-chassisd'
+
+- script: |
+    set -xe
+    python2 setup.py test
+    python2 setup.py bdist_wheel
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-ledd
+  displayName: 'Test and build sonic-ledd'
+
+- script: |
+    set -xe
+    python2 setup.py test
+    python2 setup.py bdist_wheel
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-pcied
+  displayName: 'Test and build sonic-pcied'
+
+- script: |
+    set -xe
+    python2 setup.py test
+    python2 setup.py bdist_wheel
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-psud
+  displayName: 'Test and build sonic-psud'
+
+- script: |
+    set -xe
+    python2 setup.py test
+    python2 setup.py bdist_wheel
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-syseepromd
+  displayName: 'Test and build sonic-syseepromd'
 
 - script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+    set -xe
+    python2 setup.py test
+    python2 setup.py bdist_wheel
+    python3 setup.py test
+    python3 setup.py bdist_wheel
+  workingDirectory: sonic-thermalctld
+  displayName: 'Test and build sonic-thermalctld'
+
+- publish: $(System.DefaultWorkingDirectory)/
+    artifact: sonic-platform-daemons
+    displayName: "Archive artifacts"
```

Comment:
add publish test results.

https://github.com/Azure/sonic-mgmt/blob/master/.azure-pipelines/run-test-template.yml#L88

Reply:
Working on it right now, actually :)